ext/src/ruby_api/config.rs

```diff
@@ -52,6 +59,8 @@ pub fn init() -> Result<(), Error> {
 
     class.define_singleton_method("new", function!(Config::new, 0))?;
 
+    class.define_method("consume_fuel=", method!(Config::set_consume_fuel, 1))?;
+
```

Comment:
I'm not a huge fan of these setters, I think a builder-like API would be nice here too, and coherent with `WasiCtxBuilder`. But I decided to not shave this yak just yet.